requirements.txt

```diff
@@ -0,0 +1 @@
+django
```

Comment:
This shouldn't be needed; if it was added just to satisfy a pylint check, you can disable pylint_django via pylintrc_tweaks as explained [here](https://github.com/edx/edx-lint#customizing-edx_lint).